js/adapt-stateful-session.js

```diff
@@ -127,6 +145,16 @@ define([
 			Adapt.offlineStorage.set("interaction", id, response, result, latency, responseType);
 		},
 
+		onResetSession: function () {
+			this.removeEventListeners();
+			this.setupEventListeners();
```

Comment:
I don't understand 148 + 149 - why remove then immediately re-attach the event listeners?